minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -278,6 +279,15 @@ public synchronized void stop(ServerType server, String hostname) throws IOExcep
         if (managerProcess != null) {
           try {
             cluster.stopProcessWithTimeout(managerProcess, 30, TimeUnit.SECONDS);
+            try {
+              System.setProperty("accumulo.properties",
+                  "file://" + cluster.getAccumuloPropertiesPath());
+              new ZooZap().zap(cluster.getServerContext().getSiteConfiguration(),
+                  new String[] {"-manager"});
+            } catch (Exception e) {
+              log.error("Error zapping Manager zookeeper lock", e);
+            }
```

Comment:
Setting a system property like this seems dubious. This is something a user could have set in their environment, and it's not clear the behavior if we rely on it. We should be able to call the ZooZap utility's API with the config we want to use directly, rather than relying on setting something in the environment for it to use.

Reply:
Removed system property in dc5e2c3. It wasn't needed anymore, it was needed when I was calling `ZooZap.execute` directly before I added `ZooZap.zap`